static/app/api.tsx

```diff
@@ -581,3 +585,60 @@ export class Client {
     );
   }
 }
+
+function generateOrganizationBaseUrl(organizationUrl: string) {
+  return `${organizationUrl}${DEFAULT_BASE_URL}`;
+}
+
+type APIRouteType = 'organization-events';
+
+type LegacyRoute = string;
+type CustomerDomainRoute = string;
+type RouteTuple = [LegacyRoute, CustomerDomainRoute];
+
+const routeRenderMap: Record<APIRouteType, RouteTuple> = {
+  'organization-events': ['/organizations/:org_slug/events/', '/events/'],
+};
+
+function canUseOrganizationUrl(organizationUrl: string | undefined) {
+  if (!organizationUrl) {
+    return false;
+  }
+  const currentURL = new URL('/', window.location.origin);
+  const organizationUrlParsed = new URL('/', organizationUrl);
+  return (
+    currentURL.hostname.toLowerCase() === organizationUrlParsed.hostname.toLowerCase()
+  );
```

Comment:
If the browser is on `https://acme.sentry.io` and the `organizationUrl` is `https://acme.us.sentry.io` how will this check ever pass?

Reply:
@markstory I forgot to remove this when refactoring.